optimole-wp.php

```diff
@@ -45,6 +45,10 @@ function optml_autoload( $class ) {
  */
 function optml_deactivate() {
 	if ( is_plugin_active( 'optimole-wp/optimole-wp.php' ) ) {
+		$option_name = array('optml-version', 'optml_dismissed_conflicts', 'optml_settings', 'optml_notice_optin');
+		foreach ( $option_name as $index => $option ) {
+			delete_option( $option );
+		}
```

Comment:
Can we have this in a separate function since we also use the same block of code in the `uninstall.php` ?
It would be easier to maintain if changes are required.